batch/vep/run_vep_in_variant_transforms.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# Copyright 2019 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script is intended to be used in the Docker image built for VEP. The
+# difference between this script and run_vep.sh is that it kills the VEP
+# process in case of failure or cancellation. Note that except the single
+# input, output files and the watchdog file, all other arguments are passed
+# through environment variables.
+#
+# The only environment variable that has to be set is (others are optional):
+#
```

Comment:
Would be good idea to list optional env variables. do you know why we rely on env variables in run_vep.sh? Why not directly passing them to the script?

Reply:
Since we separate the two logics, the comments have been updated.